layers/+email/gnus/packages.el

```diff
@@ -71,6 +80,16 @@
 
       (require 'browse-url)
       (require 'nnrss)
+      (defun spacemacs/gnus-flame-on ()
+        "Most important email function, for RFC1855 compliance."
+        ;; https://tools.ietf.org/html/rfc1855
+        (interactive)
+        (insert "FLAME ON:
+")
+        (insert "FLAME OFF
+")
```

Comment:
It looks a bit off to have the close quote at the beginning of the next line.
It might be better to write the newline characters as `\n`.

```elisp
        (insert "FLAME ON:\n")
        (insert "FLAME OFF\n")
```
Should there also be a semicolon after `OFF`?

Reply:
I think no, here is the [relevant page of RFC1855](https://tools.ietf.org/html/rfc1855#page-4).

Reply:
> It looks a bit off to have the close quote at the beginning of the next line.
> It might be better to write the newline characters as `\n`.
> 
> ```emacs-lisp
>         (insert "FLAME ON:\n")
>         (insert "FLAME OFF\n")
> ```
Done. Good feedback once again.